.github/workflows/integration_tests.yml

```diff
@@ -110,6 +110,13 @@ jobs:
         set -e
         echo Running test.sh
         ./src/ci/bin/test.sh
+    # This uploads the coverage report to codecov.
+    # Coverage reports are generated by SBT in src/ci/bin/test.inc.sh cromwell::private::generate_code_coverage()
+    - uses: codecov/codecov-action@v4
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        fail_ci_if_error: true # Tolerate missing codecov reports, since not all suites generate them.
```

Comment:
I think we want `false` here, the no-coverage suites are failing.

Reply:
Trying to figure out if there's a way to fail loudly if the upload fails, but also tolerate missing code-coverage for certain suites. The `handle_no_reports_found` flag does not work!

Reply:
Giving up on this for now. Will just leave `fail_ci_if_error` as `false`.